server/data/dr.strange.json.json

```diff
@@ -9,14 +9,14 @@
   "name": "Doctor Strange",
   "photoUrl": "https://i.pinimg.com/564x/1a/33/29/1a3329a57b88c0326820eca80df2a4cb.jpg",
   "schoolId": 0,
-  "schoolName": "",
-  "schoolVerified": false,
-  "shareStatistics": false,
+  "schoolName": "test school name",
+  "schoolVerified": true,
+  "shareStatistics": true,
```

Comment:
`shareStatistics` can be a simple check box (when ticked, then it's set to `true`, or `false` otherwise).